adapters/ix/ix.go

```diff
@@ -359,3 +363,33 @@ func BuildIxDiag(request *openrtb2.BidRequest) error {
 	}
 	return nil
 }
+
+// moves sid from imp[].ext.bidder.sid to imp[].ext.sid
+func moveSid(imps []openrtb2.Imp) error {
+	if len(imps) < 1 {
+		return nil
+	}
+	imp := imps[0]
+	if imp.Ext == nil || len(imp.Ext) == 0 {
```

Comment:
Nitpick: The `len` method in Go gracefully handles nil, such that the nil check isn't necessary. This can just be `iflen(imp.Ext) == 0`.